### PR TITLE
Rework helmfile values

### PR DESCRIPTION
**Pull request by dm3ch**
/close https://github.com/chainstack/graph-deployment/issues/36

### CODE REVIEW

File: helmfile/values/infra.example.yaml [hunk 1]
```diff
@@ -1,8 +1,26 @@
 # Email that would be used during issuing https certs. It would be used by let's encrypt to notify about expiring certs.
-email: example@domain.com
+email: dm3ch@dm3ch.com
```
Comment (CaJIbHuK) [inline helmfile/values/infra.example.yaml:2]: :)

**Review by CaJIbHuK — COMMENTED** [review]
I'm not sure I understand how to compose these files:
```
graphprotocol-node-index.yaml
graphprotocol-node-query.yaml
```

I see you got rid of them, but there are still defined in the helmfile

File: helmfile/values/infra.example.yaml (hunk 1 above)
Comment (dm3ch) [reply helmfile/values/infra.example.yaml:2]:
```suggestion
email: example@domain.com
```

**Comment by dm3ch** [comment]
> I'm not sure I understand how to compose these files:
> 
> ```
> graphprotocol-node-index.yaml
> graphprotocol-node-query.yaml
> ```
> 
> I see you got rid of them, but they are still defined in the helmfile

Current idea: no more secret values. Now there's just values files and we have .example.yaml files which user copies and fills before install

**Review by CaJIbHuK — APPROVED** [review]

**Pull request merged**